.travis.yml

```diff
@@ -11,11 +11,6 @@ branches:
 # by .swift-version, unless SWIFT_SNAPSHOT is specified.
 matrix:
   include:
-    - os: linux
```

Comment:
Same as PR #78 we would rather keep that for now (or have a separated PR), rather than shipping it with an unrelated PR. You can always protected specific part of your code (if needed).